src/Domain/Service/Precondition/NoAbsoluteSymlinksExistInterface.php

```diff
@@ -5,7 +5,7 @@
 /**
  * Asserts that there are no absolute symlinks in the codebase.
  *
- * This includes both the active and staging directories. Hard links are ignored.
+ * This includes both the active and staging directories. It does not apply to hard links.
```

Comment:
I actually think the previous wording was clearer. Why did it need to be changed? Maybe we can find optimal phrasing for this documentation...

Reply:
Yeah, I did it because @wimleers [felt strongly about it](https://github.com/php-tuf/composer-stager/pull/100#discussion_r1129227664). But reconsidering it, I think I agree this change is a "downgrade". I'll let @wimleers take it up with you if he still disagrees. 😉 In the interest of getting this merged, I'm going to revert it.